File: 0001-two-sum/0001-two-sum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize) 
{
    for (int i = 0; i < numsSize; i++) 
    {
        for (int j = i + 1; j < numsSize; j++) 
        {
            int *arr;
            if(nums[i]+nums[j]==target)
            {
                arr=(int *)malloc(2 * sizeof(int));
                arr[0]=i;
                arr[1]=j;
               // printf("[%d,%d]",i,j);
                *returnSize=2;
                return arr;
            }
        }
    }
    *returnSize=0;
    return NULL;
}

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> mp = new HashMap<>();
        int[] arr = new int[2];
        for(int i=0;i<nums.length;i++){
            int key = target - nums[i];
            if(mp.containsKey(key)){
                arr[0]=i;
                arr[1]=mp.get(key);
                break;
            }
            else{
                mp.put(nums[i],i);
            }
        }
        return arr;
    }
}